shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-distsql/shardingsphere-sharding-distsql-parser/src/main/java/org/apache/shardingsphere/sharding/distsql/parser/core/ShardingDistSQLStatementVisitor.java

```diff
@@ -128,6 +138,31 @@ public ASTNode visitDropShardingBindingTableRules(final DropShardingBindingTable
         return new DropShardingBindingTableRulesStatement();
     }
     
+    @Override
+    public ASTNode visitSetShardingHintDatabaseValue(final SetShardingHintDatabaseValueContext ctx) {
```

Comment:
SetShardingHintDatabaseValueContext , use verb as the very biginning of a class name, is that a specific naming rule in this satuation ?

Reply:
It is generated from a .g4 file,  It's supposed to be a special naming rule.